requirements.in

```
@@ -1,12 +1,12 @@
 bandit
-black; python_version > '3.5' and implementation_name != 'pypy'
+black; implementation_name != 'pypy'
 coverage
 flake8
-flake8-isort; python_version > '3.5'
-isort; python_version > '3.5'
+flake8-isort;
+isort;
```

I think we can remove the trailing `;` on these:

```suggestion
flake8-isort
isort
```